.pre-commit/jsonschema-dereference.py

```diff
@@ -17,56 +17,50 @@
 
 def read_yaml(file_path: Path):
     """Open and load Chart.yaml file."""
-    with open(file_path, "r") as f:
+    with open(file_path, "r", encoding="utf-8") as f:
         return yaml.load(f, Loader=Loader)
 
-def template_schema(chart_dir: Path, lock: Dict[str, Any]):
+def template_schema(chart_dir: Path, my_lock: Dict[str, Any]):
     """Load values.schema.tmpl.json and template it via Jinja2."""
-    with open(chart_dir / JSONSCHEMA_TEMPLATE_NAME, "r") as f:
-        schema_template = Template(f.read())
+    with open(chart_dir / JSONSCHEMA_TEMPLATE_NAME, "r", encoding="utf-8") as f:
+        my_schema_template = Template(f.read(), autoescape=True)
 
-    return json.loads(schema_template.render(lock))
+    return json.loads(my_schema_template.render(my_lock))
 
-def tidy_schema(schema: Any, values: Any):
+def tidy_schema(my_schema: Any, my_values: Any):
     """Hack to support OCP Form view.
 
     https://issues.redhat.com/browse/OCPBUGS-14874
     https://issues.redhat.com/browse/OCPBUGS-14875
     """
-    if isinstance(schema, dict):
-        try:
-            del schema["$schema"]
-        except:
-            pass
-        try:
-            del schema["format"]
-        except:
-            pass
+    if isinstance(my_schema, dict):
+        my_schema.pop("$schema", None)
+        my_schema.pop("format", None)
```

Comment:
.pop returns None if the key doesn't exist in the dictionary instead of throwing an exception that we have to catch and ignore